docs/source/actions.rst

```diff
@@ -300,7 +300,7 @@ very simple.
 
 You just need to follow the steps described below:
 
-1. Make sure the script comforms to the conventions described above
+1. Make sure the script conforms to the conventions described above
```

Comment:
Same :)

On Dec 1, 2015, at 6:05 AM, Tomaz Muraus <notifications@github.com<mailto:notifications@github.com>> wrote:


In docs/source/actions.rst<https://github.com/StackStorm/st2/pull/2255#discussion_r46265327>:

> @@ -300,7 +300,7 @@ very simple.
>
>  You just need to follow the steps described below:
>
> -1. Make sure the script comforms to the conventions described above
> +1. Make sure the script conforms to the conventions described above


Lol, I dared to check urban dictionary definition for "comforms" :P

�
Reply to this email directly or view it on GitHub<https://github.com/StackStorm/st2/pull/2255/files#r46265327>.